core/pom.xml

```diff
@@ -33,11 +33,67 @@
     <artifactId>mft-core</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.airavata</groupId>
+            <artifactId>mft-azure-transport</artifactId>
+            <version>0.01-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.airavata</groupId>
+            <artifactId>mft-box-transport</artifactId>
+            <version>0.01-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.airavata</groupId>
+            <artifactId>mft-gcp-transport</artifactId>
+            <version>0.01-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.airavata</groupId>
+            <artifactId>mft-local-transport</artifactId>
+            <version>0.01-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.airavata</groupId>
+            <artifactId>mft-s3-transport</artifactId>
+            <version>0.01-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.airavata</groupId>
+            <artifactId>mft-scp-transport</artifactId>
+            <version>0.01-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+         <dependency>
+            <groupId>org.apache.airavata</groupId>
+            <artifactId>mft-dropbox-transport</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```

Comment:
The mft-dropbox-transport's jar file version should ideally be 0.01 but since the convention followed is 0.0.1, I used the same here. 
@DImuthuUpe please let me know if you want me to fix this in a separate PR.

Reply:
Please fix this in a different PR. I can still see merge conflicts. 

<img width="779" alt="Screen Shot 2020-05-08 at 12 01 08 PM" src="https://user-images.githubusercontent.com/1346332/81424224-9c46f400-9123-11ea-8060-12380764f74f.png">